Sources/Datadog/Tracer.swift

```diff
@@ -139,14 +139,22 @@ public class Tracer: OTTracer {
         return nil
     }
 
+    public var activeSpan: OTSpan? {
+        return ActiveSpanUtils.getActiveSpan()
+    }
+
     // MARK: - Internal
 
     internal func createSpanContext(parentSpanContext: DDSpanContext? = nil) -> DDSpanContext {
+        var currentParentContext = parentSpanContext
+        if currentParentContext == nil {
+            currentParentContext = activeSpan?.context as? DDSpanContext
```

Comment:
if we have 2 `Tracer` objects: `Tracer1` & `Tracer2`
if the current active span is `Span1` from `Tracer1`
when we do `Tracer2.createContext(...)`, this code will take `Span1.context` as `parentContext` for `Span2`

is this the expected result?
shouldn't `Tracer1` and `Tracer2` have their own separate `activeSpan`s?

Reply:
Is it possible to have two tracers?, anyway as I said in the previous comment it must be handled by the system because it can internally transmit this information to threads when are internally created

Reply:
Yes, this will mix up the active spans, and it's also something I propose in https://github.com/DataDog/dd-sdk-ios/pull/187#discussion_r456356841 (+ this will bring less hassle in unit tests). But on the other hand, the Open Tracing standard enforces using only one tracer (`Global.sharedTracer`), so having this issue may mean API misuse, but IMO we can overcome all this with having tracer-specific active span pools.

Reply:
Ok, I implemented as a real class and assigning this object to the tracer